hack/scripts/validate_deployment.sh

```diff
@@ -0,0 +1,280 @@
+#!/bin/bash
+
+usage() {
+  cat <<EOF
+  usage: $0 [ OPTIONS ]
+
+  validate_deployment.sh will create a cluster using OCM in each of 'non-ccs' and 'ccs' types. This script should be used to test aws-account-operator deployments, and should be run after new pods are deployed to the different environments.
+
+  This script assumes that you are already logged into an OCM environment, and that you have a set of osdCcsAdmin credentials in your ~/.aws/credentials directory for the account you want to create a CCS cluster in.
+  Options
+  -e  --environment  Which OCM environment to run tests against
+  -h  --help         Show this message and exit
+  -p  --profile      Which AWS Profile should we use?
+EOF
+}
+
+# Ensure there's a parameter passed in
+if [ $# -lt 2 ]; then
+  usage
+  exit 1
+fi
+
+while [ "$1" != "" ]; do
+  case "$1" in
+    -e | --env | --environment) OCM_ENVIRONMENT=$2
+                                shift
+                                ;;
+    -h | --help )           usage
+                            exit 0
+                            ;;
+    -p | --profile )        PROFILE=$2
+                            shift
+                            ;;
+
+    * ) echo "Unexpected parameter $1"
+        usage
+        exit 1
+  esac
+  shift
+done
+
+export YELLOW="\033[33m"
+export RED="\033[31m"
+export GREEN="\033[32m"
+export RESET="\033[0m"
+
+export BULLET="\xe2\x97\x89"
+export ARROW="\xe2\x96\xb8"
+export PENDING="${YELLOW}${ARROW}${RESET}"
+export ERROR="${RED}!!${RESET}"
+export SUCCESS="${GREEN}${BULLET}${RESET}"
+
+## Define all functions, scroll to the bottom for the entrypoint of the script
+
+run_test()
+{
+  local TEST_NAME=$1
+  local TEST_CASE=$2
+  local DATETIME=$(date +%m%d)
+
+  case $TEST_NAME in
+    non-ccs)
+      local CLUSTER_NAME="aao-$DATETIME"
+      echo -e "$PENDING Creating Non-CCS Cluster $CLUSTER_NAME"
+      OCM_CREATE=$(ocm create cluster --region us-east-1 $CLUSTER_NAME)
+      if [[ $? -ne 0 ]]; then
+        echo -e "$ERROR $TEST_NAME failed.  Unable to create cluster."
+        exit 1
+      fi
+
+      # get the cluster ID from the OCM output
+      local CLUSTER_ID=$(echo $OCM_CREATE | grep '^ID:' | awk '{print $2}')
+
+      echo -e "$PENDING - Non-CCS $CLUSTER_NAME created. Cluster ID $CLUSTER_ID"
+
+      ocm_cluster_test $CLUSTER_ID
+      ;;
+    ccs)
+      local CLUSTER_NAME="aao-ccs-$DATETIME"
+      echo -e "$PENDING Creating CCS Cluster $CLUSTER_NAME"
+      eval $(get_aws_credentials)
+
+      echo -e "$PENDING Getting AWS Account Info from Profile"
+      ACCOUNT=$(aws sts get-caller-identity | jq -r .Account)
+      echo -e "$PENDING AWS Account $ACCOUNT info collected."
+
+      OCM_CREATE=$(ocm create cluster --region us-east-1 --ccs --aws-access-key-id $AWS_ACCESS_KEY_ID --aws-secret-access-key $AWS_SECRET_ACCESS_KEY --aws-account-id $ACCOUNT $CLUSTER_NAME)
```

Comment:
that is a very nice way to create a CCS cluster. I hope there is something similar to GCP as well @rafael-azevedo